tests/publiccloud/aws_cli.pm

```diff
@@ -42,7 +42,7 @@ sub run {
     my $machine_name = "openqa-cli-test-vm-$job_id";
     my $security_group_name = "openqa-cli-test-sg-$job_id";
     my $openqa_ttl = get_var('MAX_JOB_TIME', 7200) + get_var('PUBLIC_CLOUD_TTL_OFFSET', 300);
-    my $openqa_url = get_required_var('OPENQA_URL');
+    my $openqa_url = get_var('OPENQA_URL', get_var('OPENQA_HOSTNAME'));
```

Comment:
If the original line worked isn't that better?

Reply:
> If the original line worked isn't that better?

See discussion [here](https://suse.slack.com/archives/C02CANHLANP/p1689261596218609) and previous PR:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17416

Reply:
i havent followed the whole discussion on the link but i think this would look better as `get_var('OPENQA_HOSTNAME') || get_required_var( 'OPENQA_URL')`.wdyt?

Reply:
That's the same. Isn't it?

Reply:
not entirely. it is more clear and perlish IMO. when the or evaluates the second statement will return the croak error, in addition. this nested thing is weird. just my point of view